suite-native/discovery/src/discoverySelectors.ts

```diff
@@ -45,3 +71,84 @@ export const selectDiscoveryAccountsAnalytics = (
             };
         }),
     );
+
+export const selectNetworksWithUnfinishedDiscovery = (
+    state: DeviceRootState &
+        AccountsRootState &
+        FeatureFlagsRootState &
+        DiscoveryConfigSliceRootState,
+    areTestnetsEnabled: boolean,
+) => {
+    const enabledNetworkSymbols = selectEnabledDiscoveryNetworkSymbols(state);
+    const accounts = selectDeviceAccounts(state);
+    const supportedNetworks = selectDiscoverySupportedNetworks(state, areTestnetsEnabled);
+    const isCoinEnablingActive = selectIsFeatureFlagEnabled(
```

Comment:
Why checking for isCoinEnablingActive? Using enabledNetworks isn't enough?

Reply:
b675fc3e18